pkl-executor/pkl-executor.gradle.kts

```diff
@@ -49,7 +51,17 @@ sourceSets {
   }
 }
 
-tasks.test {
+val downloadPkl025 by tasks.registering(Download::class) {
+  src("https://repo1.maven.org/maven2/org/pkl-lang/pkl-config-java-all/0.25.0/pkl-config-java-all-0.25.0.jar")
+  dest("build/download/pkl-config-java-all-0.25.0.jar")
+  doFirst { file("build/download").mkdirs() }
+}
```

Comment:
[nit] this is a little nicer:

```kotlin
val pklHistoricalDistributions: Configuration by configurations.creating

dependencies {
  @Suppress("UnstableApiUsage")
  pklHistoricalDistributions(libs.pklConfigJavaAll025)
}

val prepareHistoricalDistributions by tasks.registering(Copy::class) {
  from(pklHistoricalDistributions)
  into(layout.buildDirectory.dir("pklDistributions"))
}

val prepareTest by tasks.registering {
  dependsOn(prepareHistoricalDistributions, pklDistribution)
}
```

Reply:
Ack. I went with a solution that doesn't copy. Switching to IntelliJ's Gradle test runner would simplify matters, but it's 10x slower than the already super slow platform test runner on WSL.

Reply:
Can we keep the copy solution?

This piece of code seems too brittle:

```java
        System.getProperty("user.home").toPath()
          .resolve(".gradle/caches/modules-2/files-2.1/org.pkl-lang/pkl-config-java-all/" +
            "0.25.0/e9451dda554f1659e49ff5bdd30accd26be7bf0f/pkl-config-java-all-0.25.0.jar")
```

Reply:
It's supposed to be deterministic and only used when running tests with IntelliJ's built-in test runner. But I can do a symlink instead, which also avoids a copy after every clean.

Reply:
Using IJ's JUnit run configurations have always been the better experience for me, over the Gradle run configurations. I actually like `gw prepTest` as a way to bridge the gap for us here.